pkg/controller/upload_controller.go

```diff
@@ -106,13 +106,24 @@ func (c *uploadController) enqueueUploadItem(obj interface{}) {
 
 	switch req.Status.Phase {
 	case "", pluginv1api.UploadPhaseNew, pluginv1api.UploadPhaseInProgress, pluginv1api.UploadPhaseUploadError, pluginv1api.UploadPhaseCanceling:
-		// Process New and InProgress and UploadError Uploads
+		// Process New and InProgress and UploadError and UploadCanceling Uploads
 	case pluginv1api.UploadPhaseCanceled:
 		// The upload was canceled, nothing to do.
 		log.Debug("The upload request was canceled")
 		return
+	case pluginv1api.UploadPhaseCompleted:
+		// If Upload CR status reaches terminal state more than 1 hour, Upload CR should be deleted
+		now := c.clock.Now()
+		if req.Status.CompletionTimestamp.Add(time.Hour).Unix() < now.Unix() {
+			log.Infof("Upload CR %s reaches phase %v more than 1 hour, deleting this CR.", req.Name, req.Status.Phase)
```

Comment:
Let's change this to
"Upload CR %s has been in phase %s more than 1 hour, deleting this CR."